css/main.css

```diff
@@ -0,0 +1,110 @@
+body {
+  margin: 0 0;
+  background-color: #EBEBEB;
+}
+
+p {
+  padding: 0 0;
+  margin: 0 0;
+  font-size: 1.2em;
+  text-indent: 15px;
+}
+
+li {
+  font-size: 1.2em;
+}
+
+h2 {
+  border-bottom: 1px solid #938f8f;
+}
+
+h2.first {
+  margin-left: 280px;
+}
+
+header {
+  position: fixed;
+  top: 0px;
+  width: 100%;
+  padding: 12px 0px;
+  background-color: rgb(34, 34, 34);
+}
```

Comment:
forgot make clearfix because of float buttons. Or you could use display: flex to avoid floating